### PR TITLE
Account for multiplexed samples in metadata permutation

### PR DESCRIPTION
**Pull request by jashapiro**
Here I added a couple of fields that I missed in #102.

Of course it was the multiplexed samples again...

### CODE REVIEW

File: analyses/simulate-sce/scripts/permute-metadata.R [hunk 1]
```diff
@@ -51,7 +51,9 @@ library_fields <- c(
   "tissue_spots",
   "submitter",
   "pi_name",
-  "project_title"
+  "project_title",
+  "demux_samples",
+  "demux_cell_count_estimate"
```
Comment (sjspielman) [inline analyses/simulate-sce/scripts/permute-metadata.R:56]: So, which one of these is wrong (presumably one is..?) https://github.com/AlexsLemonade/scpca-docs/blob/2ea40c44eb34a53d297edde57bc25d7eb9a8718d/docs/CHANGELOG.md?plain=1#L27

Comment (sjspielman) [reply analyses/simulate-sce/scripts/permute-metadata.R:56]: (similarly https://github.com/AlexsLemonade/scpca-docs/blob/2ea40c44eb34a53d297edde57bc25d7eb9a8718d/docs/download_files.md?plain=1#L185)

Comment (jashapiro) [reply analyses/simulate-sce/scripts/permute-metadata.R:56]: In the data I just downloaded `demux_cell_count_estimate` is the name of the field.

Comment (sjspielman) [reply analyses/simulate-sce/scripts/permute-metadata.R:56]: yup, i'll open an issue in docs

**Review by sjspielman — APPROVED** [review]

**Pull request merged**